package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-swagger",
-  "version": "4.8.3",
+  "version": "4.8.4",
```

Comment:
please revert this, we will update it when we release

Reply:
👍🏼